install/README.md

```diff
@@ -26,16 +53,31 @@ to see if your system is capable of running Kata Containers.
 > - When choosing a stable release, replace all `master` occurrences in the URLs
 > with a `stable-x.y` version available on the [download server](http://download.opensuse.org/repositories/home:/katacontainers:/releases:/).
 
-### Distros
+> **Notes on package source verification**:
+> - The Kata packages hosted on the download server are signed with GPG to ensure integrity and authenticity.
+>
+> - [This is the public key](....) used for signing the packages and [this is its fingerprint](...).
+>
+> - When prompted in the installation process, only trust these signing key and fingerprint.
+> Avoid disabling GPG checks when possible.
+
+Installing Kata Containers on your system is a three step procedure:
+1. Install the Kata Container packages.
+2. Install a supported container runtime manager.
+3. Configure the container manager to use `kata-runtime` as OCI runtime.
```

Comment:
correct me if I am wrong but we should update to use https?  And probably add a comment that suggest add/verify gpg key  with a secure channel? 
```
curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
$ sudo -E apt-get update
```

Reply:
My idea was to include the key import procedure in the installation instruction for each distribution, and to do this in a separate PR.

HTTPS will be used when importing the public key as secure channel, but it's not needed when downloading packages from OBS.

I am not sure if it's better to download the key from OBS, or have it uploaded on GitHub somewhere. Do you think it makes a difference for the tests infrastructure?

Reply:
having the key import done in the instructions means that this will also be run with tests,
and we can drop the `--no-gpg-checks` flags  used now.

Reply:
Nice. It's one extra step, but having the key import step clearly stated will make it clearer to users what is happening from a security perspective.